docs/extending/developing-code/extending/case-studies/Curl/basic.rst

```diff
@@ -0,0 +1,80 @@
+Basic version of ``Curl`` using Rules
+=====================================
+
+Armed with information from the previous sections:
+
+* where the function belongs: "Mathematical Operations", and
+* information from WMA and SymPy on what should be implemented and how it hooks into an existing library function, and
+* an implementation of Curl written in Mathics to guide us
+
+we are now ready to being with a skeletal version from which we can fill out later.
+
+"Mathematical Operations" is located in directory ``mathics/builtin/vectors/math_ops.py``.
```

Comment:
Directory or file?

Reply:
You are correct. It should be "file". Changed in 4689c81